aspnetcore/tutorials/first-mvc-app/adding-view.md

```diff
@@ -14,71 +14,93 @@ By [Rick Anderson](https://twitter.com/RickAndMSFT)
+In the **Add New Item - MvcMovie** dialog:
 
-  * Select **Add**
+* In the search box in the upper-right, enter *view*
+* Select **Razor View**
+* Keep the **Name** box value, *Index.cshtml*.
+* Select **Add**
 
-![Add New Item dialog](adding-view/_static/add_view.png)
+![Add New Item dialog](adding-view/_static/add_view50.png)
 
 # [Visual Studio Code](#tab/visual-studio-code)
 
-Add an `Index` view for the `HelloWorldController`.
+Add an `Index` view for the `HelloWorldController`:
 
 * Add a new folder named *Views/HelloWorld*.
 * Add a new file to the *Views/HelloWorld* folder name *Index.cshtml*.
 
 # [Visual Studio for Mac](#tab/visual-studio-mac)
 
-* Right click on the *Views* folder, and then **Add > New Folder** and name the folder *HelloWorld*.
-* Right click on the *Views/HelloWorld* folder, and then **Add > New File**.
-* In the **New File** dialog:
+Control-click on the *Views* folder, and then **Add > New Folder** and name the folder *HelloWorld*.
 
-  * Select **ASP .NET Core** in the left pane.
-  * Select **Razor View** in the center pane.
-  * Type *Index* in the **Name** box.
-  * Select **New**.
+Control-click on the *Views/HelloWorld* folder, and then **Add > New File**.
 
-![Add New Item dialog](adding-view/_static/add_view_macVSM8.9.png)
+In the **New File** dialog:
+
+* Select **ASP.NET Core** in the left pane.
+* Select **Razor View - Empty** in the center pane.
+* Type *Index* in the **Name** box.
+* Select **New**.
+
+  ![Add New Item dialog](adding-view/_static/add_view_macVSM8.9.png)
 
 ---
 
 Replace the contents of the *Views/HelloWorld/Index.cshtml* Razor view file with the following:
 
 [!code-cshtml[](~/tutorials/first-mvc-app/start-mvc/sample/MvcMovie22/Views/HelloWorld/Index1.cshtml?highlight=7)]
 
-Navigate to `https://localhost:{PORT}/HelloWorld`. The `Index` method in the `HelloWorldController` didn't do much; it ran the statement `return View();`, which specified that the method should use a view template file to render a response to the browser. Because a view template file name wasn't specified, MVC defaulted to using the default view file. The default view file has the same name as the method (`Index`), so the view template in */Views/HelloWorld/Index.cshtml* is used. The image below shows the string "Hello from our View Template!" hard-coded in the view.
+Navigate to `https://localhost:{PORT}/HelloWorld`:
 
-![Browser window](~/tutorials/first-mvc-app/adding-view/_static/hell_template.png)
+* The `Index` method in the `HelloWorldController` ran the statement `return View();`, which specified that the method should use a view template file to render a response to the browser.
+* A view template file name wasn't specified, MVC defaulted to using the default view file. The default view file has the same name as the method (`Index`), so the view template in */Views/HelloWorld/Index.cshtml* is used.
+* The following image shows the string "Hello from our View Template!" hard-coded in the view:
```

Comment:
hard-coded is slang and doesn't MT. It's not needed here, they see where it comes from. This only makes sense once you explain dynamic values.